README.md

```diff
@@ -35,14 +35,16 @@ Docker Compose installation options.
 
     docker_apt_release_channel: stable
     docker_apt_arch: amd64
-    docker_apt_repository: "deb [arch={{ docker_apt_arch }}] https://download.docker.com/linux/{{ ansible_distribution|lower }} {{ ansible_distribution_release }} {{ docker_apt_release_channel }}"
+    docker_apt_repository: "deb [arch={{ docker_apt_arch }}] https://download.docker.com/linux/{{ ansible_distribution | lower }} {{ ansible_distribution_release }} {{ docker_apt_release_channel }}"
     docker_apt_ignore_key_error: True
+    docker_apt_gpg_key: https://download.docker.com/linux/{{ ansible_disztribution | lower }}/gpg
```

Comment:
Can you please document these two new variables (`docker_apt_gpg_key` and `docker_yum_gpg_key`) in the README in the appropriate section?

Also, `ansible_disztribution` is misspelled here.

Reply:
Hi. Thank you for reviewing this. Sorry for my mistake. I hopefully fixed it in your suitable expectations. :)